common/test/python/projects/test_study.py

```diff
@@ -65,7 +65,7 @@ def test_apply(self):
                         center_id="dummy",
                         adcid=1)
         center.apply(visitor)
-        assert visitor.center_name == "Dummy Center"
+        assert visitor.center_id == "Dummy Center"
```

Comment:
This test fails - `common/src/python/projects/study.py` needs to update the `Study.apply` method to say `visitor.visit_center(self.name)` instead of `visitor.visit_center(self)`

Reply:
Fixed. The error was in the Center.apply method